ui/src/app/pages/data/concept/concept-set-details.tsx

```diff
@@ -204,10 +254,11 @@ export const ConceptSetDetails =
         <div style={{display: 'flex', flexDirection: 'column'}}>
           <div style={styles.conceptSetHeader}>
             <div style={{display: 'flex', flexDirection: 'row'}}>
-              <ConceptSetMenu canDelete={workspacePermissions.isOwner}
-                              canEdit={workspacePermissions.canWrite}
+              <ConceptSetMenu canDelete={workspace.accessLevel === WorkspaceAccessLevel.OWNER}
```

Comment:
nit: I thought there was a `canDelete` helper somewhere around here. This comparison is more logic than I like to see inside html; one level of abstraction higher like `workspace.canDelete` would be more readable and future-proof I think.

Reply:
there's a `WorkspacePermissions` class you can create to get this if you have the entire `Workspace` and `WorkspaceAccessLevel`. I could change it to only need the `WorkspaceAccessLevel` for this function and just use that.